### PR TITLE
t0062 cleanup

### PR DESCRIPTION
**Pull request by chriscool**
These are just clean ups to make sure that everything is tested and to avoid potential quoting problems.

### CODE REVIEW

**Comment by whyrusleeping** [comment]
this LGTM

File: test/sharness/t0062-daemon-api.sh [hunk 1]
```diff
@@ -12,20 +12,21 @@ test_init_ipfs
...
+	api_fromcfg=$(ipfs config Addresses.API) &&
+	peerid=$(ipfs config Identity.PeerID)
+'
 
 test_client() {
-	args="$@"
-	printf $peerid >expected
-	ipfs $args id -f="<id>" >actual
+	printf "$peerid" >expected &&
+	ipfs "$@" id -f="<id>" >actual &&
```
Comment (jbenet) [inline test/sharness/t0062-daemon-api.sh]: we only use one flag now, but if there's more, this will break

Comment (chriscool) [inline test/sharness/t0062-daemon-api.sh]: @jbenet are you sure?
"$@" was made to properly deal with arguments:

```
$ test_args() { for a in "$@"; do echo "a: '$a'"; done; }
$ test_args "arg 1" arg2 "arg3" '  arg 4 '
a: 'arg 1'
a: 'arg2'
a: 'arg3'
a: '  arg 4 '
```

Comment (jbenet) [inline test/sharness/t0062-daemon-api.sh]: ah, interesting. i didn't test it. i assumed from expectations about strings. ok

**Pull request merged**